script.js

```diff
@@ -88,20 +88,21 @@ const Weathertron = {
 		let html = "";
 		console.log(simpleForecast);
 
-		for (let i = 0; i < simpleForecast.length; i++) {
-			const fc = simpleForecast[i];
-			const iconSrc = imgUrl + fc.icon + ".png";
+		// for (let i = 0; i < simpleForecast.length; i++)
+		simpleForecast.forEach((element, idx) => {
+			const fc = simpleForecast[idx];
```

Comment:
No need to do this assignment, you could have made the `element` argument `fc` instead! That's the whole point of forEach. No need for idx.